<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckUI</class>
 <widget class="QWidget" name="CheckUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(12, 58, 152);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>241</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>72</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>ATM</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DepositButton">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>340</y>
     <width>470</width>
     <height>200</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>72</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(1, 180, 245);</string>
   </property>
   <property name="text">
    <string>예금</string>
   </property>
  </widget>
  <widget class="QPushButton" name="WithdrawpushButton">
   <property name="geometry">
    <rect>
     <x>1300</x>
     <y>340</y>
     <width>470</width>
     <height>200</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>72</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(1, 180, 245);</string>
   </property>
   <property name="text">
    <string>출금</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SendButton">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>560</y>
     <width>470</width>
     <height>200</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>72</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(1, 180, 245);</string>
   </property>
   <property name="text">
    <string>송금</string>
   </property>
  </widget>
  <widget class="QPushButton" name="QuitButton">
   <property name="geometry">
    <rect>
     <x>1300</x>
     <y>560</y>
     <width>470</width>
     <height>200</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>72</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(44, 118, 255);</string>
   </property>
   <property name="text">
    <string>취소</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>370</y>
     <width>431</width>
     <height>374</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="namelabel">
      <property name="font">
       <font>
        <pointsize>36</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>예금주명</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="namelineEdit">
      <property name="styleSheet">
       <string notr="true">font: 28pt &quot;Sans Serif&quot;;
background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="UIDlabel">
      <property name="font">
       <font>
        <pointsize>36</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>계좌번호</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="UIDlineEdit">
      <property name="styleSheet">
       <string notr="true">font: 28pt &quot;Sans Serif&quot;;
background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="budgetlabel">
      <property name="font">
       <font>
        <pointsize>36</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>잔고</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="budgetlineEdit">
      <property name="styleSheet">
       <string notr="true">font: 28pt &quot;Sans Serif&quot;;
background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
